<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraphics</class>
 <widget class="QWidget" name="ConfigureGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="toggle_vsync">
          <property name="text">
           <string>Enable V-Sync</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_bos">
          <property name="text">
           <string>Ignore format reinterpretation</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="toggle_frame_limit">
            <property name="text">
             <string>Limit Speed Percent</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="frame_limit">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Renderer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Backend</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="renderer_comboBox">
          <item>
           <property name="text">
            <string>Software</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OpenGL</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="software_render_group" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="software_shader_jit">
           <property name="text">
            <string>Enable Shader JIT</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="opengl_render_group" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label2">
               <property name="text">
                <string>Shader Emulation</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="hw_shaders">
               <item>
                <property name="text">
                 <string>CPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GPU</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Internal Resolution:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="resolution_factor_combobox">
               <item>
                <property name="text">
                 <string>Auto (Window Size)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Native (400x240)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2x Native (800x480)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3x Native (1200x720)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4x Native (1600x960)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5x Native (2000x1200)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6x Native (2400x1440)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7x Native (2800x1680)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8x Native (3200x1920)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9x Native (3600x2160)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10x Native (4000x2400)</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="accurate_hardware_shader">
             <property name="text">
              <string>Accurate Hardware Shader</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="opengl_shader_jit">
             <property name="text">
              <string>Enable Shader JIT</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="layoutBox">
     <property name="title">
      <string>Layout</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label2">
            <property name="text">
             <string>Background color:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="layout_bg">
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label1">
            <property name="text">
             <string>Screen Layout:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="layout_combobox">
            <item>
             <property name="text">
              <string>Default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Single Screen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Large Screen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Side by Side</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="swap_screen">
          <property name="text">
           <string>Swap Screens</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
